templates/renew-script.sh.erb

```diff
@@ -1,5 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
`metadata.json` says the module work on some non-linux systems where you can't count on `bash(1)` (and if available is not in `/bin`). So this script should be kept portable.

```suggestion
#!/bin/sh
```

Reply:
Fixed.